database_docs/doc_base.json

```diff
@@ -1,404 +1,404 @@
+    "differential_enthalpy_fixed_dry_bulb_high_limit_shutoff_dry_bulb_temp": "Dry-bulb temperature high limit shutoff for differential enthalpy and fixed dry-bulb economizer control",
+    "differential_enthalpy_fixed_dry_bulb_is_allowed": "Is differential enthalpy and fixed dry-bulb economizer control allowed?",
+    "differential_enthalpy_is_allowed": "Is differential enthalpy economizer control type allowed?",
+    "dirt_correction_factor_for_solar_and_visible_transmittance": "Direct correction factor for solar and visible transmittance",
+    "draft_type": "Draft type",
+    "draw_profile": "Hot water draw profile associated with the water heater rating",
+    "electric_equipment_average_epd": "Average electric equipment power density for the space",
+    "electric_equipment_epd_unit": "Electric equipment power density unit",
+    "electric_equipment_fraction_latent": "Electric equipment power density latent fraction",
+    "electric_equipment_fraction_lost": "Electric equipment power density lost fraction",
+    "electric_equipment_fraction_radiant": "Electric equipment power density radiant fraction",
+    "electric_equipment_maximum_epd": "Maximum electric equipment power density",
+    "electric_equipment_median_epd": "Median electric equipment power density",
+    "electric_equipment_minimum_epd": "Minimum electric equipment power density",
+    "electric_equipment_space_type_name": "Electric equipment space type name",
+    "electric_power_phase": "Power phase",
+    "electronic_enthalpy_is_allowed": "Is electronic enthalpy economizer control allowed?",
+    "end_date": "Requirement applicability end date",
+    "energy_factor_base": "Energy factor base coefficient, the energy factor requirement is calculated as follows: energy_factor_base + volume * energy_factor_volume_derate",
+    "energy_factor_volume_derate": "Energy factor volume derating coefficient, the energy factor requirement is calculated as follows: energy_factor_base + volume * energy_factor_volume_derate",
+    "energy_recovery_effectiveness": "Energy recovery effectiveness (unitless)",
+    "enthalpy_recovery_ratio": "Enthalpy recovery ratio as per AHRI Standard 1060",
+    "equipment_type": "Type of equipment",
+    "exit_lighting_schedule": "Schedule to use for exit lights",
+    "fan_cooling_application": "Fan cooling application",
+    "fan_type": "Cooling tower fan type",
+    "fixed_dry_bulb_high_limit_shutoff_temp": "Dry-bulb temperature high limit shutoff for fixed dry-bulb economizer control",
+    "fixed_dry_bulb_is_allowed": "Is fixed dry-bulb temperature economizer control allowed?",
+    "fixed_enthalpy_fixed_dry_bulb_high_limit_shutoff_dry_bulb_temp": "Enthalpy high limit shutoff for fixed enthalpy and fixed dry-bulb economizer control",
+    "fixed_enthalpy_fixed_dry_bulb_high_limit_shutoff_enthalpy": "Dry-bulb temperature high limit shutoff for fixed enthalpy and fixed dry-bulb economizer control",
```

Comment:
@lymereJ I think you need to switch the two descriptions.

Reply:
Good catch!